packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -33,7 +33,9 @@ then
 fi
 
 echo "- Run local network"
-startInBgAndWaitForString 'Ganache STARTED' yarn devchain run-tar packages/protocol/$BUILD_DIR/devchain.tar.gz >> $LOG_FILE
+yarn devchain run-tar-in-bg packages/protocol/$BUILD_DIR/devchain.tar.gz >> $LOG_FILE
+# TODO (soloseng): fix `startInBgAndWaitForString` PIPE error
+# startInBgAndWaitForString 'Ganache STARTED' yarn devchain run-tar packages/protocol/$BUILD_DIR/devchain.tar.gz >> $LOG_FILE
```

Comment:
The previous PIPE method of running ganache in background is no longer working with the current version. 

Since it was only used in 1 script, I have replaced it using ganache builtin detach mode.